api/test/venues.test.ts

```diff
@@ -72,26 +65,26 @@ const regularOpeningDays = [
 
 const exceptionalClosedDays = [
   {
-    overrideDate: "2024-01-01T00:00:00.000Z",
-    type: "Christmas and New Year",
+    overrideDate: "2024-04-05T00:00:00.000Z",
+    type: "Free bank holiday weekend!",
     startDateTime: "00:00",
     endDateTime: "00:00",
   },
   {
-    overrideDate: "2024-03-29T00:00:00.000Z",
-    type: "Easter",
+    overrideDate: "2024-04-06T00:00:00.000Z",
+    type: "Free bank holiday weekend!",
     startDateTime: "00:00",
     endDateTime: "00:00",
   },
   {
-    overrideDate: "2024-03-30T00:00:00.000Z",
-    type: "Easter",
+    overrideDate: "2024-04-08T00:00:00.000Z",
+    type: "Free bank holiday weekend!",
     startDateTime: "00:00",
     endDateTime: "00:00",
   },
   {
-    overrideDate: "2024-03-31T23:00:00.000Z",
-    type: "Easter",
+    overrideDate: "2024-04-30T23:00:00.000Z",
+    type: "Fête du travail",
```

Comment:
🇫🇷

Reply:
Could I get more context on changing the dates/how it affects other tests and why the type changed as well?

Reply:
I only changed the dates to give us extra reassurance that the `exceptionalClosedDays` are in fact removed from the API response `nextOpeningDays` list. The previous `exceptionalClosedDays` didn't fall within the timeframe we were requesting
The type comes from Prismic originally and doesn't affect the result